Example/Podfile

```diff
@@ -13,6 +13,14 @@ target 'ExampleApp' do
   use_frameworks!
 
   # Pods for ExampleApp
-  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.26.1"
+  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.27.0"
   pod 'Kingfisher'
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+    end
+  end
+end
```

Comment:
Why is this needed? 
Will this be also required now for the client integration?

Reply:
Yes, it is now needed, because of the XCFramework. That is one of the reasons we need updated documentation.